api/http/routing.go

```diff
@@ -63,7 +63,8 @@ const (
 	ApiUrlManagementLimitsName = ApiUrlManagement + "/limits/#name"
 
 	ApiUrlManagementV2               = "/api/management/v2/deployments"
-	ApiUrlManagementV2ReleaseTags    = ApiUrlManagementV2 + "/releases/#name/tags"
+	ApiUrlManagementV2ReleasesName   = ApiUrlManagementV2 + "/deployments/releases/#name"
+	ApiUrlManagementV2ReleaseTags    = ApiUrlManagementV2ReleasesName + "/tags"
```

Comment:
Not sure if a `PUT /releases/#name/notes` would be more appropriate than an incomplete `PATCH /releases/#name`?

Reply:
I prepared it so we can easily in the future update other fields, so I kindly suggest to keep it as is.